src/palettes/palette_file_map.json

```diff
@@ -3,5 +3,6 @@
   "My Family Palette": "myfamily",
   "People": "people",
   "BMW Palette": "bmw_palette",
-  "Palettes": "palettes"
+  "Palettes": "palettes",
+  "InputArea": "input_area"
```

Comment:
The use of "InputArea" and "input_area" don't match with "outputArea" and "output_area.json" used elsewhere. Fortunately, the output area is only rendered once at the instantiation so this bug doesn't cause any problem. :)

It seems reasonable either calling this area an input or an output. I personally lean towards "input" as it's an area to receive user inputs. :)

Reply:
Good point, @cindyli .  If this was not bliss, it would a standard text _input_ field for entering text.  I'll switch it around to consistently use "input" in the code and json.  But I won't rename the branch nor the issue.

Reply:
> But I won't rename the branch nor the issue.

I agree.

Reply:
Fixed.  Ready for another review, thanks.